infra/scripts/replace-running-components.sh

```diff
@@ -99,7 +99,7 @@ function check-name-isnt-ambiguous ()  {
 # Translates VCH name to ID if necessary
 function get-vic-id () {
     if [[ -z $VIC_ID ]]; then
-        export VIC_ID="$($VIC_DIR/bin/vic-machine-linux ls --target=$target --user=$username --password=$password --thumbprint=$(get-thumbprint) | grep $VIC_NAME | awk '{print $1}')"
```

Comment:
thank you so much for all the places where you put quotes around variables like on this line -- this is something I should've done and know better etc.